source/events/interaction-events.md

```diff
@@ -65,25 +77,32 @@ Detail:
 - `worldPosition`: a THREE.js Vector3 of the hover world position
 - `position`: a THREE.js Vector3 of the local hover position
 
-## `touch`
+## `touchmove`
+
+Emitted continuously while the trigger/pinch or direct touch collision is maintained
 
 Origin: the target element
 
 Detail:
 
 - `worldPosition`: a THREE.js Vector3 of the hover world position
 - `position`: a THREE.js Vector3 of the local hover position
-- `delta`: a THREE.js Vector3 of the delta between the current local position and the last
+- `delta`: a THREE.js Vector3 of the delta between the the current local position and the last
 
-??? do we still use this?
+## `touchend`
 
-## `touchmove`
+Triggered whenever the controller's trigger button or pinch gesture is released, or when the user stops touching an entity.
+
+Origin: the target element
 
-???
+---
 
+# Supported MouseEvents (in XR and 2D)
 
-## `touchend`
+These are standard MouseEvents and _should_ behave as they would on any webpage.
 
-Triggered whenever the controller's trigger button or pinch gesture is released, or when the user stops touching an entity.
+## `click`
 
-Origin: the target element
+## `mouseover`
+
+## `mouseout`
```

Comment:
this should probably be its own event page instead of being hidden at the bottom - just adding the file in the folder directly should update the section on the left so there's nothing more involved to do other than that

Reply:
also make sure to add the same sections like you did for the above interaction items

description

origin:

detail:
- ...

Reply:
Ah then I might leave these undocumented, we don't add any more detail to them than the standard MouseEvents implementation.

Reply:
But we will in a future update when I refactor the ControlSystem

Reply:
sounds good -then this isnt a pr blocker, just remove from this pr and make an open issue in the documentation repo as a reminder for when we get to that ~